src/index.js

```diff
@@ -68,6 +69,7 @@ export default {
   List,
   ListItem,
   Segment,
+  Segments,
```

Comment:
As of [this `Statistic` PR conversation](https://github.com/TechnologyAdvice/stardust/pull/117/files#r45690951), we've begun adding the plural grouping class as a sub-component of the singular component.  Have a look there and let me know what you think.